packages/sdks/web-component/src/lib/helpers/templates.ts

```diff
@@ -276,16 +298,16 @@ const setImageVariable = (
 export const updateTemplateFromScreenState = (
   baseEle: DocumentFragment,
   screenState?: ScreenState,
-  componentsConfig?: ComponentsConfig,
   flowInputs?: Record<string, string>,
   logger?: { error: (message: string, description: string) => void },
 ) => {
   replaceHrefByDataType(baseEle, 'totp-link', screenState?.totp?.provisionUrl);
   replaceHrefByDataType(baseEle, 'notp-link', screenState?.notp?.redirectUrl);
   replaceElementTemplates(baseEle, screenState);
-  setElementConfig(baseEle, componentsConfig, logger);
+  setElementConfig(baseEle, screenState?.componentsConfig, logger);
```

Comment:
what is this change for?

Reply:
There’s no need to pass `componentsConfig` separately because it’s already included as part of screenState